lib/OpenQA/WebAPI/Controller/API/V1/JobGroup.pm

```diff
@@ -96,11 +96,14 @@ sub find_group {
     return $group;
 }
 
+has _cached_properties => undef;
```

Comment:
Private attribute accessors are not very useful, using `$self->{cached_properties}` is much more efficient. But if you really want to use one it should be declared at the top of the file.

Reply:
Changed to `$self->{cached_properties}`.

Reply:
So it seems you are no longer using the accessor, but forgot to delete the definition?

Reply:
Yes. I've removed it now.